catalogs/catalog-lakehouse-paimon/build.gradle.kts

```diff
@@ -30,29 +30,36 @@ val sparkMajorVersion: String = sparkVersion.substringBeforeLast(".")
 val paimonVersion: String = libs.versions.paimon.get()
 
 dependencies {
-  implementation(project(":api"))
-  implementation(project(":common"))
-  implementation(project(":core"))
+  implementation(project(":api")) {
+    exclude("*")
+  }
+  implementation(project(":common")) {
+    exclude("*")
+  }
+  implementation(project(":core")) {
+    exclude("*")
+  }
   implementation(libs.bundles.paimon) {
     exclude("com.sun.jersey")
     exclude("javax.servlet")
+    exclude("org.apache.hive")
+    exclude("org.apache.hbase")
+    exclude("it.unimi.dsi")
+    exclude("org.mortbay.jetty")
+    exclude("org.apache.parquet")
+    exclude("org.mortbay.jetty:jetty")
+    exclude("org.mortbay.jetty:jetty-util")
+    exclude("org.mortbay.jetty:jetty-sslengine")
+    exclude("org.eclipse.jetty.aggregate:jetty-all")
   }
   implementation(libs.bundles.log4j)
   implementation(libs.commons.lang3)
-  implementation(libs.caffeine)
   implementation(libs.guava)
   implementation(libs.hadoop2.common) {
     exclude("com.github.spotbugs")
     exclude("com.sun.jersey")
     exclude("javax.servlet")
-  }
-  implementation(libs.hadoop2.hdfs) {
```

Comment:
@LiuQhahah gravitino paimon catalog supports FilesystemCatalog as its backend catalog, and it depend on the hdfs dependency.

Reply:
could you please do a basic test locally? such as built successfully and ran through the unit tests.